CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 * `Menu` now outputs semantic HTML. Links are rendered in HTML lists, with submenus rendered with nested lists.
 
+## Bug Fixes
+
+* Fixed (https://jira-sage.valiantyscloud.net/browse/GOLF-53): When selecting all in selectable grids with a secondary theme, the odd lined items were displaying a white background showing as being broken.
```

Comment:
This release note is not in an open source context. It's a good idea to raise an issue and link to that, or just make the release note generic. Also, I'd call it a table rather than a grid in this context

Reply:
By making it generic should I remove the linked Jira ticket?

Reply:
👍 It references a private resource.